README.md

```diff
@@ -12,39 +11,53 @@ Clone this repository
 git clone https://github.com/opentransportmanager/otm-api.git
 ```
 
-Install necessary dependencies
+Run docker containers from the project root:
 ```
-composer install
+docker-compose up -d
 ```
+It will auto-install required dependencies, create matching .env file with new application key and migrate the database 
 
-In the root of a project copy .env.example into .env
+Done! Your application should be by default accesible at:
 ```
-cp .env_example .env
+http://localhost:8000
```

Comment:
Port 8000 is a really common thing. It would be nice if you could provide a way to configure this port.